app/browser/reducers/tabsReducer.js

```diff
@@ -291,6 +352,14 @@ const tabsReducer = (state, action, immutableAction) => {
         tabs.setTabIndex(action.get('tabId'), action.get('index'))
       })
       break
+    case appConstants.APP_TAB_SET_FULL_SCREEN: {
+      const isFullscreen = action.get('isFullScreen')
+      const tabId = action.get('tabId')
+      if (isFullscreen === true || isFullscreen === false) {
```

Comment:
If the tab is already `FullScreen`, do we need to set it to full screen again?

Reply:
This code was moved in https://github.com/brave/browser-laptop/pull/13783/commits/23f5b719fd8ee0a610e8d52685602f4fc37221f7. At this point, I believe that the tab content area is not fullscreen, but other UX may be (e.g. the window). Agreed that if that's the case, perhaps `isFullscreen` is not the best variable name here, although it's more of a state description - "This tab is in fullscreen mode or not, so tell the tab's content to go fullscreen too, or not)".